samples/msteams-virtual-assistant/csharp/readme.md

```diff
@@ -0,0 +1 @@
+Please refer to the [Virtual Assistant Template documentation](http://aka.ms/virtualassistantdocs) for deployment and customization instructions.
```

Comment:
Do you have instructions to run this sample locally? Could you please add those or link to specific page where developers can find these steps?

Reply:
updated the link